packages/synapse-react-client/src/components/SynapseTable/SynapseTableConstants.ts

```diff
@@ -1,6 +1,8 @@
+import { IconName } from '../IconSvg'
+
 export const TOOLTIP_DELAY_SHOW = 500
 export const SELECT_ALL = 'SELECT_ALL'
 export const DESELECT_ALL = 'DESELECT_ALL'
 export const NOT_SET_DISPLAY_VALUE = '\u2013' // html dash
 // double check these icons!
-export const ICON_STATE: string[] = ['sortdown', 'sortdown', 'sortup']
+export const ICON_STATE: IconName[] = ['sortDown', 'sortDown', 'sortUp']
```

Comment:
Just needed to change the type since `IconSvg` uses a union of string literals (i.e. TypeScript makes sure that we only use strings that map to icons, you can't just use any string you want)

Reply:
Thanks for the explanation Nick! Is there any reason why sortDown gets repeated two times?

Reply:
There are three sort states for a given column - null (no sort), descending, and ascending. We use the same icon for null and descending (applying a color change when necessary). The original implementer decided to use an array to represent these states.